core/chains/evm/client/errors.go

```diff
@@ -178,7 +178,20 @@ var harmony = ClientErrors{
 	Fatal:                   harmonyFatal,
 }
 
-var clients = []ClientErrors{parity, geth, arbitrum, optimism, metis, substrate, avalanche, nethermind, harmony, besu, erigon}
+var clients = []ClientErrors{parity, geth, arbitrum, optimism, metis, substrate, avalanche, nethermind, harmony, besu, erigon, klaytn}
+
+// Klaytn
+// https://github.com/klaytn/klaytn/blob/dev/blockchain/error.go
+// https://github.com/klaytn/klaytn/blob/dev/blockchain/tx_pool.go
+var klaytn = ClientErrors{
+	NonceTooLow:                 regexp.MustCompile(`(: |^)nonce too low$`),
+	TransactionAlreadyInMempool: regexp.MustCompile(`(: |^)(known transaction|there is another tx which has the same nonce in the tx pool$)`), // `known transaction` error could end with additional information
```

Comment:
Isn't this reserved for exact matches? If there is _another_ tx with the same nonce, then wouldn't this be either "nonce too low" or "replacement underpriced"?

Reply:
Klaytn uses this sentence instead of `"replacement underpriced"`. It's a legacy of Klaytn's unit price policy. At the beginning of Klaytn, Klaytn only allows a specific gas price (unit price) to mitigate the hardness of gas estimation. That's why Klaytn has a different sentence. 
Recently, Klaytn has adopted a dynamic price, but it is slightly different from Ethereum's. BTW, it is using the same old sentence still.

Reply:
@aidan-kwon : Can you confirm the exact meaning/behavior of the error message "there is another tx which has the same nonce in the tx pool"?

Does it mean that the server is rejecting this current Tx, since another different one already exists?
As @jmank88 said, we normally map that case to ReplacementTransactionUnderpriced.

Reply:
Actually just looking at the code for this error here: https://github.com/klaytn/klaytn/blob/dev/blockchain/tx_pool.go#L916

This is exactly the ReplacementTransactionUnderpriced case. You are rejecting this Tx if it isn't a higher gas price than previous one.

Reply:
Thank you @prashantkumar1982 @dimriou @jmank88 
Yes, you are right. `there is another tx which has the same nonce in the tx pool` is the case of `ReplacementTransactionUnderpriced`. I mapped the message to `ReplacementTransactionUnderpriced`.